src/components/Products/Products.js

```diff
@@ -24,18 +25,62 @@ const Products = ({ item }) => {
       console.log("error");
```

Comment:
Suggestions: Console logs can be avoided. It can be used while development for testing purposes